codemods/styled-components-to-ucl.ts

```diff
@@ -92,10 +93,19 @@ export function transformStyledCompoentsToUCL(
     })
     .closest(j.TaggedTemplateExpression)
     .forEach((nodePath) => {
-      // @ts-ignore
+      if (
+        !(
+          nodePath.node.tag.type === "MemberExpression" &&
+          nodePath.node.tag.property.type === "Identifier"
+        )
+      ) {
+        // TODO: Replace node with mocked component, log manual work?
+        throw new Error("Error #jj43");
```

Comment:
arbitrary error. I don't think we'll ever hit this, but its possible we have some edge case im not aware of yet. If we hit this we'll mock the component and log manual work